outlines/react.md

```diff
@@ -0,0 +1,34 @@
+# React
+
+1. Introduction to React
+  1. Background: developed by Facebook, used widely in production, vibrant ecosystem
+  2. Installing and using React in Meteor 1.3
```

Comment:
Can we have something about how to use a 3rd party react component also? (I actually have no idea how to do this now in 1.3, and would love to know)